lib/Connection.php

```diff
@@ -28,6 +30,12 @@ class Connection
     /** @var string */
     private $uri;
 
+    /** @var string */
+    private $password = '';
```

Comment:
Password should default to `null` IMO, does Redis allow empty passwords?

Reply:
[Documentation](https://redis.io/commands/auth) says nothing about an empty password, let's assume it's not allowed.